onnxscript/rewriter/pattern.py

```diff
@@ -23,21 +23,96 @@
 # appropriate).
 
 
-class ConstantPattern:
-    def __init__(self, value: int | str | list) -> None:
+class PythonPattern:
```

Comment:
nit: Why `PythonPattern`? Not that it matters, since this is outside the scope of this PR. We will need to clean the pattern IR next, separately.

Reply:
@titaiwangms

Reply:
Referring to python type: int/str/float, etc. And these are not going to be serialized.